tests/unit/NameRequestInfo.spec.ts

```diff
@@ -10,8 +10,8 @@ const vuetify = new Vuetify({})
 const store = getVuexStore() as any // remove typings for unit tests
 
 const localVue = createLocalVue()
-localVue.filter("VMask", data => data) // mock filter
-localVue.filter("capitalize", data => data) // mock filter
+localVue.filter('VMask', data => data) // mock filter
+localVue.filter('capitalize', data => data) // mock filter
```

Comment:
Ah, oops, thanks for the lint cleanup :)

Reply:
It's done automatically. 😄